app/components/webauthn_input_component.rb

```diff
@@ -0,0 +1,20 @@
+class WebauthnInputComponent < BaseComponent
+  attr_reader :platform, :tag_options
+
+  alias_method :platform?, :platform
+
+  def initialize(platform: false, **tag_options)
+    @platform = platform
+    @tag_options = tag_options
+  end
+
+  def call
+    content_tag(
+      :'lg-webauthn-input',
+      content,
+      **tag_options,
+      hidden: true,
```

Comment:
There's some discussion about making the inputs disabled rather than hidden when not supported, which will require us to change up the implementation. I think I'd also like to refactor what exists currently duplicated between [`_mfa_selection.html.erb`](https://github.com/18F/identity-idp/blob/9498cb76bfe5ab091563ee180a92cc4a238a4f7e/app/views/partials/multi_factor_authentication/_mfa_selection.html.erb) and [`two_factor_authentication/index.htmlerb`](https://github.com/18F/identity-idp/blob/9498cb76bfe5ab091563ee180a92cc4a238a4f7e/app/views/two_factor_authentication/options/index.html.erb#L19-L36) to a new "Checkbox Input Component", at which point it'd be reasonable for that to be rendered by this component, and we could pass the `disabled` option that way.